src/pages/lib/auth/manage-session/q/platform/[platform].mdx

```diff
@@ -33,64 +33,66 @@ Before you begin you will need:
 
 ## Retrieve your current authenticated user
 
-You can use the `Auth.currentAuthenticatedUser()` API which returns the full `CognitoUser` object. This object contains information about the currently authenticated user including the session and user attributes.
-
-You can call `Auth.currentAuthenticatedUser()` and get the current authenticated user object:
+You can use the `getCurrentUser` API to get information about the currently authenticated user including the `username`, `userId` and `signInDetails`.
 
 <BlockSwitcher>
 <Block name="TypeScript">
 
 ```ts
-import { Auth } from 'aws-amplify';
+import { getCurrentUser } from "aws-amplify/auth";
 
 async function currentAuthenticatedUser() {
   try {
-    const user = await Auth.currentAuthenticatedUser({
-      bypassCache: false // Optional and is false by default. If set to true, this call 
-      // will send a request to Cognito to get the latest user data.
-    });
-    console.log(user);
-  } catch(err) {
+    const user = await getCurrentUser();
+    console.log(`The username: ${user.username}`);
+    console.log(`The userId: ${user.userId}`);
+    console.log(`The signInDetails: ${user.signInDetails}`);
```

Comment:
Nit: this would be more succinct destructuring as you've done elsewhere, then we wouldn't need the log statements.
```suggestion
    const { username, userId, signInDetails } = await getCurrentUser();
```